addons/uav/XEH_postInit.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+
+["Air", "init", { // on air vehicle init, add action to class if has pylons
+    params ["_vehicle"];
+    _isUAV = unitIsUAV _vehicle;
+    _hasPilotCamera = hasPilotCamera _vehicle;
+    if(_isUAV && _hasPilotCamera) then {
+        _pfhID = [{
+            params ["_args", "_pfID"];
+            _args params ["_vehicle"];
+
+            private _pilotCameraTarget = getPilotCameraTarget _vehicle;
+            private _pilotCameraRotation = getPilotCameraRotation _vehicle;
+            if (_pilotCameraTarget select 0) then {
+                private _pilotCameraTargetPos = _pilotCameraTarget select 1;
+                private _pilotCameraTargetRelPos = vectorNormalized ((_vehicle worldToModel (ASLToAGL _pilotCameraTargetPos)) vectorDiff (getPilotCameraPosition _vehicle));
+                _pilotCameraRotation = [ -(rad ((_pilotCameraTargetRelPos select 0) atan2 (_pilotCameraTargetRelPos select 1)) ), -(rad asin(_pilotCameraTargetRelPos select 2))]
+            };
+            _vehicle animateSource ["mainTurret", _pilotCameraRotation select 0];
+            _vehicle animateSource ["mainGun", -(_pilotCameraRotation select 1)];
```

Comment:
How does this not break any third party UAV with pilot camera?

Reply:
Because, as I understand, the `animateSource` command does nothing unless the animation source is set to 'user'. It is also my understanding that if that's the case, the gunner position cannot actually control the turret. Ergo, if the above are a true, a 3rd party UAV can be broken by this command, its turret cannot be operated by a gunner, and if its turret can be operated by a gunner, then it cannot by broken by this command.

Reply:
So you run this loop despite it doing nothing on those vehicles. That's terrible.